.env.in

```diff
@@ -1,15 +1,5 @@
-#### ref ######
-REDIS_URI=your-redis.redis.cache.windows.net:[port],password=,ssl=True,abortConnect=False
-KURS_PM_API_KEY=[secret key to connect to kurs-pm-api]
-KURS_PM_API_URI=[Available at Azure Secrets page]
-KOPPS_URI=[Available at Azure Secrets page]
-SESSION_SECRET=[something random]
-SESSION_KEY=kurs-pm.pid
-OIDC_APPLICATION_ID=<FROM ADFS>
-OIDC_CLIENT_SECRET=<FROM ADFS>
-OIDC_TOKEN_SECRET=<Random string>
-BLOB_SERVICE_SAS_URL=https://kursinfostoragestage.blob.core.windows.net/?sv=[date]&ss=b&srt=o&sp=rwcx&se=[date]&st=[date]&spr=https&sig=[generated signature]
-STORAGE_CONTAINER_NAME=memo-blob-container
-/*If you want to start your server on another port, add the following two variables, else use default ones from serversettings.js*/
-# SERVER_PORT=[your port for the server]
-# SERVER_HOST_URL=http://localhost:[SERVER_PORT]
\ No newline at end of file
+KURS_PM_API_KEY=[Available in Azure KeyVault]
```

Comment:
I'm not sure if we need `SESSION_SECRET` (and should set it to something random?) 
What do you think, @karlandindrakryggen ?

Reply:
Perfect!